lib/routes/account.js

```diff
@@ -1146,6 +1142,10 @@ module.exports = (log, db, mailer, Password, config, customs, signinUtils, push)
       method: 'POST',
       path: '/account/destroy',
       config: {
+        auth: {
+          mode: 'optional',
+          strategy: 'sessionToken'
```

Comment:
To keep backwards compatibility, made `sessionToken` auth optional.

Reply:
This will be necessary for at least a train, but is the intention to eventually make it required?  I don't expect anything other than content-server to be calling this endpoint I don't think, so we could evolve towards that if desired.